lib/easy-build.js

```diff
@@ -54,14 +64,39 @@ module.exports = {
    * @public
    *
    * @param {object} grunt - grunt instance.
+   * @param {JSON} userTasks - dict with customized downstream tasks
```

Comment:
I have not seen jsdoc utilize the `JSON` type before. Usually just define it as an `object`.... And actually I believe it should be `Object` rather than `object`. 

Not a huge fan of their documentation but for reference: http://usejsdoc.org/tags-type.html